Install documentation -x.
Fix html links by including section number in target.

diff -r d802a4e69c72 Documentation/Makefile
--- a/Documentation/Makefile	Thu Jul 27 19:52:54 2006 -0700
+++ b/Documentation/Makefile	Thu Jul 27 19:59:48 2006 -0700
@@ -50,14 +50,14 @@ txt7: $(MAN7_TXT)
 
 install: man txt
 	$(INSTALL) -m755 -d $(DESTDIR)/$(txtdir)
-	$(INSTALL) $(MAN1_TXT) $(MAN7_TXT) $(DESTDIR)/$(txtdir)
+	$(INSTALL) -m644 $(MAN1_TXT) $(MAN7_TXT) $(DESTDIR)/$(txtdir)
 	$(INSTALL) -m755 -d $(DESTDIR)/$(man1) $(DESTDIR)/$(man7)
-	$(INSTALL) $(DOC_MAN1) $(DESTDIR)/$(man1)
-	$(INSTALL) $(DOC_MAN7) $(DESTDIR)/$(man7)
+	$(INSTALL) -m644 $(DOC_MAN1) $(DESTDIR)/$(man1)
+	$(INSTALL) -m644 $(DOC_MAN7) $(DESTDIR)/$(man7)
 
 install-html: html
 	$(INSTALL) -m755 -d $(DESTDIR)/$(htmldir)
-	$(INSTALL) $(DOC_HTML) $(DESTDIR)/$(htmldir)
+	$(INSTALL) -m644 $(DOC_HTML) $(DESTDIR)/$(htmldir)
 
 test:
 	make -C tutorial-script test
diff -r d802a4e69c72 Documentation/asciidoc.conf
--- a/Documentation/asciidoc.conf	Thu Jul 27 19:52:54 2006 -0700
+++ b/Documentation/asciidoc.conf	Thu Jul 27 19:59:48 2006 -0700
@@ -17,7 +17,7 @@ endif::backend-docbook[]
 
 ifdef::backend-xhtml11[]
 [gitlink-inlinemacro]
-<a href="{target}.html">{target}{0?({0})}</a>
+<a href="{target}{0?.{0}}.html">{target}{0?({0})}</a>
 endif::backend-xhtml11[]
 
 [attributes]
